cosmoz-grouped-list.js

```diff
@@ -274,7 +279,7 @@ export class CosmozGroupedList extends templatizing(PolymerElement) {
 
 			// groups with items
 			if (item.items.length) {
-				return acc.concat(item, item.items);
+				return acc.concat(item, item.items.map(i => Object.assign(i, { [symbols.group]: item })));
```

Comment:
Using this symbol to keep track of what group an item belongs to. That way we don't need to loop thru data also to find a item's group.
We should probably add a `items` symbol too but we'll check later in omnitable.